source/codegen/metadata/nirfmxnr/functions.py

```diff
@@ -2951,7 +2951,9 @@
         'init_method': True,
         'parameters': [
             {
+                'cross_driver_session': 'ViSession',
```

Comment:
I expect that these are already being applied in scrapigen and just not copied over yet. Is that what's happening? if not, something's wrong with the rfmxshared.transform code that applies this.

Reply:
Hmm, I'm not seeing it in scrapigen's [functions.py](https://github.com/ni/grpc-device-scrapigen/blob/main/out/rfmxnr/export/functions.py#L2949). Built locally too and it looks the same. I'll try to figure it out and get it generating the same for all personalities.

Reply:
Oops, NR needs to call [tag_cross_driver_sessions ](https://github.com/search?q=tag_cross_driver_sessions&type=code) and so do the other personalities. I missed that in the shuffle.

Reply:
I have a [PR in scrapigen](https://github.com/ni/grpc-device-scrapigen/pull/28) up to address this for the other personalities. The updated metadata will be pulled into grpc-device as we add the cross-driver session test per personality.